### ISSUE TITLE
Expected Identifier - IE8 Issue with user.js

### ISSUE THREAD

**Issue by jasonflaherty** [Pomax/Usered#2]
I have been trying out the simple user registration and ran into the error with a login:

Expected Identifier
user.js     Line: 21
Code: 0   Char:12

The demo setup shows this error also. I have been looking into a fix, but haven't found one yet. 

FYI. Thanks for a great simple system!

**Comment by jasonflaherty** [comment]
I am not sure if this is a "fix", but I get a different error down the script when I add this to change how Line: 21 is showing: 

```
    markInvalid: function(input, reason) {
    var classes = "";
    var c = "class"; 
    if(input.c) { classes = input.getAttribute("class"); }
```

I guess IE reserves the word class, so the var c solves that?

The next error that shows is:

Expected Identifier, string or number
Line: 201 Char:30

```
add(form, make("label", {for: "usered_username", innerHTML: "user name"}));
```

I don't see any additional commas or quotes or anything... Not sure about this one. However, if I take out the "FOR" the error subsides...

```
add(form, make("label", {innerHTML: "user name"}));
```

IE8 will still not allow the user to enter the folder where the "secure" content is being held. 

Maybe not issue, since IE8 is phasing out, but thought I'd share.

**Comment by Pomax** [comment]
yeah, the problem is the reserved keyword "class", which should be a quoted property access rather than direct access. I'll fix this later today, thanks for pointing it out!

So, rather than what you did, if(input["class"]) { ... } -- your solution actually looks for a property called "c" on the input element, which it won't find, so the code in the conditional block won't run.

**Comment by Pomax** [comment]
https://github.com/Pomax/Usered/commit/6227574435e235423c15abbd1d368d0b782514bb and https://github.com/Pomax/Usered/commit/5df1a716a2adc2fd722b4a0d56e3620880752c28 should have fixed this issue, let me know if you're still experiencing problems with the updated code.
